### PR TITLE
Improve launch script

### PR DESCRIPTION
**Pull request by tanaes**
This adds capability to easily launch a 'wet' test run, simply by doing:

```bash
oecophylla workflow --test
```

You can also provide other workflow parameters to test specific things, like testing a particular module:

```bash
oecophylla workflow --test qc
```

or testing in a particular workflow context:

```bash
oecophylla workflow --workflow-type torque --cluster-config cluster.json --test qc
```

what do you think @mortonjt? I went back and forth a ton of times on how best to do this, but this seemed most parsimonious.

### CODE REVIEW

File: oecophylla/cli/launch.py [hunk 1]
```diff
@@ -72,8 +83,8 @@ def _create_dir(_path):
               type=click.Choice(['torque', 'slurm', 'local']),
               default='local',
               help='Select where to run the pipeline (cluster or locally).')
-@click.option('--output-dir', '-o', type=click.Path(),
-              help='Input directory of all of the samples.')
+@click.option('--output-dir', '-o', type=click.Path(), required=False,
```
Comment (mortonjt) [inline oecophylla/cli/launch.py:111]: What's the default if this isn't specified?

File: oecophylla/cli/launch.py [hunk 2]
```diff
@@ -49,18 +49,29 @@ def _create_dir(_path):
...
+    else:
+        oeco_dir = os.path.abspath(d)
+        return oeco_dir
+
+
 @run.command()
 @click.argument('targets', nargs=-1)
 #TODO add an option to process multiple directories
-@click.option('--input-dir', '-i', required=True, type=click.STRING,
+@click.option('--input-dir', '-i', required=False, type=click.STRING,
```
Comment (mortonjt) [inline oecophylla/cli/launch.py:91]: Same here.  What happens if this isn't specified?

File: oecophylla/cli/launch.py (hunk 1 above)
Comment (mortonjt) [reply oecophylla/cli/launch.py:111]: Oh, nvm. This is to allow for testing scenarios

File: oecophylla/cli/launch.py (hunk 2 above)
Comment (mortonjt) [reply oecophylla/cli/launch.py:91]: Clarified - this is to allow for testing scenarios

File: oecophylla/cli/launch.py [hunk 3]
```diff
@@ -86,67 +97,118 @@ def _create_dir(_path):
               help='Restarts the run and overwrites previous input.')
 @click.option('--just-config', is_flag=True, flag_value=False,
               help='Only generate the configuration file.')
+@click.option('--test', is_flag=True, default=False,
+              help='Executes a run with the included test data.')
 def workflow(targets, input_dir, sample_sheet, params, envs,
```
Comment (mortonjt) [inline oecophylla/cli/launch.py:127]: Can you update the Readme on how to test run the workflow?

**Review by mortonjt — COMMENTED** [review]
Given how many changes are being introduced to enable the testing parameter, I'd consider moving it to a separate function - for readability and modular testing. But I'll leave this decision to you

File: oecophylla/cli/launch.py (hunk 3 above)
Comment (tanaes) [reply oecophylla/cli/launch.py:127]: Sure thing.

**Comment by tanaes** [comment]
@mortonjt: good ideas, implemented.

**Review by mortonjt — APPROVED** [review]
Looks good!

**Comment by tanaes** [comment]
@mortonjt @tkosciol wanna merge?

**Pull request merged**